catalog/alfresco/alfresco.bom

```diff
@@ -186,8 +186,7 @@ brooklyn.catalog:
             ALFRESCO_BASEUNIXSERVICE_SCRIPT_NAME: $brooklyn:config("alfresco.base.unix.service.script.name")
             ALFRESCO_SERVICES_STARTUP: $brooklyn:config("alfresco.services.startup")
           install.command: |
-            sudo yum update -y
-            sudo yum install -y augeas java-1.8.0-openjdk
+            sudo yum install -y curl augeas java-1.8.0-openjdk
```

Comment:
@aledsage Any particular reason you removed `sudo yum update -y`? Thought it was a best practice to perform an `yum update` before `yum install`

Reply:
Explanation was buried inside a commit message:
```
Remove `yum update` from install command

It should not be the responsibility of the blueprint to do this.
If a user has a pre-defined AMI with particular versions of packages
which they’ve been using in QA/UAT, then we should not upgrade them
all automatically.
```

Do you think that's a good reason? I believe that doing the `yum install` will be a no-op if any version of the given package is already installed (i.e. for curl, augeas and java-1.8.0-openjdk).

Reply:
@aledsage Ah sorry, I haven't ready the commit messages...

That's interesting, I see the point your making and it makes sense. Although, it is a radical change from the way we are handling things up until now: all blueprints I can think of do `yum update`. Removing it means that the blueprint will either require a custom AMI, or some manual step to do `yum update`. It may not be a bad thing, I'm just pointing out that it's a big shift. If we go down that route, I think all other blueprints should do the same for consistency.

WDYT @aledsage ?

Reply:
I don't think we'd need a custom AMI. We still run `yum install` to ensure the things you need are installed. If you choose an old AMI then it will use the old packages - you can test it, and then hopefully know that it will keep working. Just now, a blueprint that works on a given AMI might stop working in the future because of a `yum update`.

I agree we want a more consistent approach, but that approach might just need to be consistent per user. A user could include a hardening script (e.g. an additive `pre.install`), whereas if you include `yum update` then one needs to replace the entire `install` command to change that behaviour.

Do you want me to remove that change from the PR, and submit it separately for more discussion?

Reply:
Nah, it's fine for this PR, don't worry.

But we definitely to keep it in mind and inform our users of this